setup.cfg

```diff
@@ -16,7 +16,7 @@ doc_files = Changelog
     examples/
 
 [flake8]
-max-line-length = 99
+max-line-length = 120
```

Comment:
I am a friend of PEP-0008, which advocates a [limit of 99 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length). Thus I would prefer to keep this value as it is, if you do not see good reasons for a change.